exiftool.go

```diff
@@ -95,6 +101,12 @@ func (e *Exiftool) Close() error {
 		errs = append(errs, fmt.Errorf("error while closing stdin: %w", err))
 	}
 
+	if e.cmd != nil {
+		if err := e.cmd.Wait(); err != nil {
```

Comment:
Do you know if `e.cmd.Wait()` can block ? I'm wondering if a "timer" should be added so that `Close()` function can return, even if anything blocks `e.cmd.Wait()`.
What do you think about it ?

Reply:
[os.exec.Cmd.Wait()](https://pkg.go.dev/os/exec#Cmd.Wait) calls [os.Process.Wait()](https://pkg.go.dev/os#Process.Wait) which [blocks](https://cs.opensource.google/go/go/+/master:src/os/exec_unix.go). Would you rather hide the blocking nature of `Close()` or expose that to users? e.g. users can use their own timers to timeout calling `Close()` instead of us doing that for them by timing out `Wait()`

Reply:
I think that go-exiftool should be, from the user point of vue, as easy to use as possible. Could you handle the timeout ?